### ISSUE TITLE
Fix Image Alignment on First Run Page

### ISSUE THREAD

**Issue by bensternthal** [mozilla/bedrock#5643]
Per discussion w. Jenny (and Ben), this card is blocked pending the results for the upcoming vertical alignment test - https://github.com/mozilla/bedrock/issues/5641

**Comment by ejregithub** [comment]
@alexgibson - we didn't really get one, just 'doesn't look properly aligned'. For now, this remains blocked on the results of #5644, which will likely remove the need for this task.

**Comment by alexgibson** [comment]
> @alexgibson - we didn't really get one, just 'doesn't look properly aligned'. For now, this remains blocked on the results of #5644, which will likely remove the need for this task.

Ok understood. Do we know who made the initial request? If we can find out what they mean exactly that would help. This could also be an easy fix potentially, so it may not need be blocked by #5644 (which could take some time).

**Comment by alexgibson** [comment]
Given that this page is moving in-product, I'd say we wontfix this and close the issue/bug?

**Comment by ejregithub** [comment]
Closing as invalid - this page is going in-product
